sources/packages/backend/workflow/test/2023-2024/parttime-assessment/parttime-assessment-2023-2024-eligibility-CSGP.e2e-spec.ts

```diff
@@ -0,0 +1,48 @@
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-CSGP.`, () => {
+  it(
+    "Should determine CSGP as eligible when total assessed need is greater than 0 " +
+      "and application PD/PPD status is true for a single student.",
```

Comment:
If a single student is not affecting the E2E scenario we can refrain from having it added to the description.

Reply:
https://github.com/bcgov/SIMS/pull/2506#discussion_r1394937155